packages/flutter_test/lib/src/finders.dart

```diff
@@ -178,6 +178,20 @@ class CommonFinders {
   Finder byElementPredicate(ElementPredicate predicate, { String description, bool skipOffstage: true }) {
     return new _ElementPredicateFinder(predicate, description: description, skipOffstage: skipOffstage);
   }
+
+  /// Looks for widgets that match the pattern of descendant finder under the
+  /// widget tree with ancestor as the root.
+  ///
+  /// Example:
+  ///
+  ///     expect(tester, hasWidget(find.descedant(
+  ///       ancestor: find.widgetWithText(Row, 'label_1'), descendant: find.text('value_1')));
+  ///
+  /// If the `skipOffstage` argument is true (the default), then this skips
+  /// nodes that are [Offstage] or that are from inactive [Route]s.
+  Finder descendant({ Finder ancestor, Finder descendant, bool skipOffstage: true }) {
```

Comment:
you don't document here that ancestor has to only match one node. That seems like a weird limitation though. Why not allow ancestor to match multiple nodes?